tests/pester-tests/commands/choco-list.Tests.ps1

```diff
@@ -37,6 +37,26 @@ Describe "choco list" -Tag Chocolatey, ListCommand {
         }
     }
 
+    Context "Listing packages ignoring pinned packages" {
+        BeforeAll {
+            $null = Invoke-Choco pin add --name="installpackage"
+
+            $Output = Invoke-Choco list --ignore-pinned
+        }
+
+        It "Exits with Success (0)" {
+            $Output.ExitCode | Should -Be 0 -Because $Output.String
+        }
+
+        It "Should contain non-pinned packages" {
+            $Output.Lines | Should -Contain "upgradepackage 1.0.0"
+        }
+
+        It "Should not contain pinned packages" {
+            $Output.Lines | Should -Contain "installpackage 1.0.0"

```

Comment:
This test is wrong.

The test expects the `installpackage` to be listed, but in this case it shouldn't be listed.